xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1834 @@
+   <para>
+     They will continue to operate correctly under &productname; &productnumber; &clm;,
+     but we recommend that you migrate to using the newer <literal>designate-worker</literal>
+     <literal>designate-producer</literal> service components instead by
+     following the procedure documented in <xref linkend="DNS-MIGRATE"/>.
+   </para>
+
+   <procedure>
+     <title>Cleanup Orphaned Packages</title>
+     <step>
+       <para>
+         After migrating the deployer node, there are a small number of
+         packages that were installed that are no longer required&mdash;such as
+         the <systemitem class="service">ceilometer</systemitem> and <systemitem class="service">freezer</systemitem> <literal>virtualenv</literal> (venv) packages.
+       </para>
+       <para>
+         You can safely remove these packages with the following command:
+       </para>
+       <screen>&prompt.ardana;zypper packages --orphaned
+Loading repository data...
+Reading installed packages...
+S | Repository | Name                             | Version                           | Arch
+--+------------+----------------------------------+-----------------------------------+-------
+i | @System    | python-flup                      | 1.0.3.dev_20110405-2.10.52        | noarch
+i | @System    | python-happybase                 | 0.9-1.64                          | noarch
+i | @System    | venv-openstack-ceilometer-x86_64 | 9.0.8~dev7-12.24.2                | noarch
+i | @System    | venv-openstack-freezer-x86_64    | 5.0.0.0~xrc2~dev2-10.22.1         | noarch
+ardana> sudo zypper remove venv-openstack-ceilometer-x86_64 venv-openstack-freezer-x86_64
+Loading repository data...
+Reading installed packages...
+Resolving package dependencies...
+
+The following 2 packages are going to be REMOVED:
+  venv-openstack-ceilometer-x86_64 venv-openstack-freezer-x86_64
+
+2 packages to remove.
+After the operation, 79.0 MiB will be freed.
+Continue? [y/n/...? shows all options] (y): y
+(1/2) Removing venv-openstack-ceilometer-x86_64-9.0.8~dev7-12.24.2.noarch ..................................................................[done]
+Additional rpm output:
+/usr/lib/python2.7/site-packages/ardana_packager/indexer.py:148: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
+  return yaml.load(f)
+
+
+(2/2) Removing venv-openstack-freezer-x86_64-5.0.0.0~xrc2~dev2-10.22.1.noarch ..............................................................[done]
+Additional rpm output:
+/usr/lib/python2.7/site-packages/ardana_packager/indexer.py:148: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
+  return yaml.load(f)
+       </screen>
+     </step>
+   </procedure>
+
+   <procedure>
+     <title>Delete freezer Containers from the Swift Object Store</title>
```

Comment:
Should the "Swift Object Store" in this title line be using the entity reference?

Reply:
Ahh, the &o_objstore; one?

Not being familiar with OpenStack, I am still learning what abbreviation stands for which component.